.python-version

```diff
@@ -0,0 +1 @@
+3.13.5
```

Comment:
Nit: do we need it? I would tend to think just relying on `pyproject.toml` `requires-python` is less error-prone.

Reply:
Hm, not strictly, what it does is make sure everyone's on the same Python version when developing with UV. Whether that's desirable or not, I have arguments either way. 😆